tests/factory/test_factory_scraper.py

```diff
@@ -1,44 +1,62 @@
 import copy
 import os
 
-import mozfile
 import pytest
 
 from mozdownload import FactoryScraper
 
 
-@pytest.mark.parametrize('scraper_type,builds,fname,arg', [
+@pytest.mark.parametrize('filename,args', [
```

Comment:
Why did you change the number of arguments? Note that `args` is used for the arguments as passed to the scraper, but not to push a dict into the test method. You want to revert this specific change, so that it is identical with the other tests.

Reply:
I changed it based on this (https://github.com/mozilla/mozdownload/pull/537#discussion_r258910087) discussion.  Please correct me if i misunderstood something.

Reply:
Sorry, I only specified part of the string but not how the whole arguments list should look like. To clarify I meant `scraper_type,builds,filename,args`.